bazel/repository_locations.bzl

```diff
@@ -227,6 +226,24 @@ REPOSITORY_LOCATIONS = dict(
         # 2019-07-17
         urls = ["https://github.com/bazelbuild/rules_foreign_cc/archive/a209b642c7687a8894c19b3dd40e43e6d3f38e83.tar.gz"],
     ),
+    rules_cc = dict(
```

Comment:
are these now required by protobuf 3.9.1 bump or? I don't think they are released yet.

Reply:
They are:

https://github.com/protocolbuffers/protobuf/blob/v3.9.1/protobuf_deps.bzl#L25-L47

Reply:
@jmillikin-stripe do you know the status of those rules? just wanted to make sure how we track updates here.

Reply:
I don't know how close these are to having a numbered point release. The closest thing they have to "releases" appears to be ad-hoc copying of particular Git archives into Bazel's mirror. Maybe we should use those pseudo-release versions here?

rules_proto (from https://github.com/bazelbuild/rules_proto/blob/master/README.md):
```python
    rules_proto = dict(
        sha256 = "602e7161d9195e50246177e7c55b2f39950a9cf7366f74ed5f22fd45750cd208",
        strip_prefix = "rules_proto-97d8af4dc474595af3900dd85cb3a29ad28cc313",
        # 2019-08-02
        urls = [
            "https://mirror.bazel.build/github.com/bazelbuild/rules_proto/archive/97d8af4dc474595af3900dd85cb3a29ad28cc313.tar.gz",
            "https://github.com/bazelbuild/rules_proto/archive/97d8af4dc474595af3900dd85cb3a29ad28cc313.tar.gz",
    
        ],
    ),
```

rules_cc (from https://github.com/bazelbuild/rules_proto/blob/9cd4f8f1ede19d81c6d48910429fe96776e567b1/proto/private/dependencies.bzl):
```python
    rules_cc = dict(
        sha256 = "bb8320b0bc1d8d01dc8c8e8c50edced8553655c03776960c1287d03dfbcac3e5",
        strip_prefix = "rules_cc-401380cd2279b83da0dcb86ecbac04a04805405b",
        # 2019-08-09
        urls = [
            "https://mirror.bazel.build/github.com/bazelbuild/rules_cc/archive/401380cd2279b83da0dcb86ecbac04a04805405b.tar.gz",
            "https://github.com/bazelbuild/rules_cc/archive/401380cd2279b83da0dcb86ecbac04a04805405b.tar.gz",
        ],
    ),
```

rules_java (from https://github.com/bazelbuild/rules_proto/blob/9cd4f8f1ede19d81c6d48910429fe96776e567b1/proto/private/dependencies.bzl):
```python
    rules_java = dict(
        sha256 = "4e2f33528a66e3a9909910eaa5a562fb22f5b422513cdc3816fd01fbb6e2d08d",
        strip_prefix = "rules_java-166a046a27e118d578127759b413ee0b06aca3cd",
        # 2019-08-07
        urls = [
            "https://mirror.bazel.build/github.com/bazelbuild/rules_java/archive/166a046a27e118d578127759b413ee0b06aca3cd.tar.gz",
            "https://github.com/bazelbuild/rules_java/archive/166a046a27e118d578127759b413ee0b06aca3cd.tar.gz",
        ],
    ),
```

Reply:
Done, please take a look.